DQMOffline/Trigger/python/HiggsMonitoring_cfi.py

```diff
@@ -0,0 +1,96 @@
+)
+hltHIGmonitoring.histoPSet.ptPSet = cms.PSet(
+  nbins = cms.int32 (  60   ),
+  xmin  = cms.double(   0   ),
+  xmax  = cms.double(  300  ),
+)
+hltHIGmonitoring.histoPSet.phiPSet = cms.PSet(
+  nbins = cms.int32 (  32  ),
+  xmin  = cms.double( -3.2 ),
+  xmax  = cms.double(  3.2 ),
+)
+hltHIGmonitoring.histoPSet.etaPSet = cms.PSet(
+  nbins = cms.int32 (  30  ),
+  xmin  = cms.double( -3.0 ),
+  xmax  = cms.double(  3.0 ),
+)
+hltHIGmonitoring.histoPSet.htPSet = cms.PSet(
+  nbins = cms.int32 (   60  ),
+  xmin  = cms.double(   0   ),
+  xmax  = cms.double(  600  ),
+)
+
+hltHIGmonitoring.applyleptonPVcuts = cms.bool(True)
+hltHIGmonitoring.leptonPVcuts = cms.PSet(
+  dxy = cms.double(   0.5  ),
+  dz  = cms.double(   1.   ),
+)
+
+
+#MET and HT binning
+hltHIGmonitoring.histoPSet.metBinning = cms.vdouble(0,20,40,60,80,100,125,150,175,200)
+hltHIGmonitoring.histoPSet.HTBinning  = cms.vdouble(0,20,40,60,80,100,125,150,175,200,300,400,500,700)
+#Eta binning
+hltHIGmonitoring.histoPSet.eleEtaBinning = cms.vdouble(-2.4,-2.1,-1.7,-1.2,-0.9,-0.6,-0.3,-0.1,0,0.1,0.3,0.6,0.9,1.2,1.7,2.1,2.4)
+hltHIGmonitoring.histoPSet.jetEtaBinning = cms.vdouble(-2.4,-2.1,-1.7,-1.2,-0.9,-0.6,-0.3,-0.1,0,0.1,0.3,0.6,0.9,1.2,1.7,2.1,2.4)
+hltHIGmonitoring.histoPSet.muEtaBinning  = cms.vdouble(-2.4,-2.1,-1.7,-1.2,-0.9,-0.6,-0.3,-0.1,0,0.1,0.3,0.6,0.9,1.2,1.7,2.1,2.4)
+#pt binning
+hltHIGmonitoring.histoPSet.elePtBinning = cms.vdouble(0,3,5,8,10,15,20,25,30,40,50,60,80,120,200,400)
+hltHIGmonitoring.histoPSet.jetPtBinning = cms.vdouble(0,3,5,8,10,15,20,25,30,40,50,60,80,120,200,400)
+hltHIGmonitoring.histoPSet.muPtBinning  = cms.vdouble(0,3,5,8,10,15,20,25,30,40,50,60,80,120,200,400)
+#Eta binning 2D
+hltHIGmonitoring.histoPSet.eleEtaBinning2D = cms.vdouble(-2.4,-1.7,-1.2,-0.6,-0.3,0,0.3,0.6,1.2,1.7,2.4)
+hltHIGmonitoring.histoPSet.jetEtaBinning2D = cms.vdouble(-2.4,-1.7,-1.2,-0.6,-0.3,0,0.3,0.6,1.2,1.7,2.4)
+hltHIGmonitoring.histoPSet.muEtaBinning2D  = cms.vdouble(-2.4,-1.7,-1.2,-0.6,-0.3,0,0.3,0.6,1.2,1.7,2.4)
+#pt binning 2D
+hltHIGmonitoring.histoPSet.elePtBinning2D = cms.vdouble(0,15,20,30,40,60,80,100,200,400)
+hltHIGmonitoring.histoPSet.jetPtBinning2D = cms.vdouble(0,15,20,30,40,60,80,100,200,400)
+hltHIGmonitoring.histoPSet.muPtBinning2D  = cms.vdouble(0,15,20,30,40,60,80,100,200,400)
+#HT and phi binning 2D
+hltHIGmonitoring.histoPSet.HTBinning2D  = cms.vdouble(0,20,40,70,100,150,200,400,700)
+hltHIGmonitoring.histoPSet.phiBinning2D = cms.vdouble(-3.1416,-2.5132,-1.8849,-1.2566,-0.6283,0,0.6283,1.2566,1.8849,2.5132,3.1416)
+
+
+hltHIGmonitoring.met       = cms.InputTag("pfMetEI") # pfMet
+hltHIGmonitoring.jets      = cms.InputTag("pfJetsEI") # ak4PFJets, ak4PFJetsCHS
+hltHIGmonitoring.electrons = cms.InputTag("gedGsfElectrons") # while pfIsolatedElectronsEI are reco::PFCandidate !
+hltHIGmonitoring.muons     = cms.InputTag("muons") # while pfIsolatedMuonsEI are reco::PFCandidate !
+hltHIGmonitoring.vertices  = cms.InputTag("offlinePrimaryVertices")
```

Comment:
couldn't you use the goodPrimaryVertices collection ?
I think it should be run w/in RECO step, but I might be wrong,
please check